sklearn/preprocessing/_encoders.py

```diff
@@ -0,0 +1,888 @@
+# Authors: Andreas Mueller <amueller@ais.uni-bonn.de>
+#          Joris Van den Bossche <jorisvandenbossche@gmail.com>
+# License: BSD 3 clause
+
+from __future__ import division
+
+import numbers
+import warnings
+
+import numpy as np
+from scipy import sparse
+
+from ..base import BaseEstimator, TransformerMixin
+from ..externals import six
+from ..utils import check_array
+from ..utils.fixes import _argmax
+from ..utils.validation import check_is_fitted, FLOAT_DTYPES
+from .label import LabelEncoder
+
+
+BOUNDS_THRESHOLD = 1e-7
```

Comment:
Where is this used?

Reply:
sorry, that's a left-over from copying the code from `data.py`, not sure why my linter didn't catch that. Will remove.